### PR TITLE
[src&tests] Improve scheduler's docs + add plot method

### PR DESCRIPTION
**Pull request by mpariente**
- `plot` can be useful for the lazy ones (me included)
- It was a bit undiscoverable till now. It's still pretty undiscoverable but at least we mention it in the docs.

### CODE REVIEW

File: asteroid/engine/schedulers.py [hunk 1]
```diff
@@ -128,3 +146,12 @@ def _get_lr(self):
...
         return lr
+
+    def as_tensor(self, start=0, stop=None):
+        stop = stop or self.warmup_steps * 2
+        lr_list = []
+        for _ in range(start, stop):
+            self.step_num += 1
+            lr_list.append(self._get_lr())
+        self.step_num = 0
+        return torch.tensor(lr_list)
```
Comment (mpariente) [inline asteroid/engine/schedulers.py:157]:
```suggestion
```

**Comment by popcornell** [comment]
Looks good to me, pretty useful addition !!
